### ISSUE TITLE
🐛 Bug: When selecting option "Remove type annotations that don't change the meaning of code", it will turn into "Add missing property declarations to classe"

### ISSUE THREAD

**Issue by rubiesonthesky** [JoshuaKGoldberg/TypeStat#1486]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/TypeStat/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

Selecting option "Remove type annotations that don't change the meaning of code" for "Which improvements would you like to make?" would stay as "Remove type annotations"

### Actual

Selecting option "Remove type annotations that don't change the meaning of code" for "Which improvements would you like to make?", turns into "Add missing property declarations to classes". I'm not sure if this is what is supposed to happen?

```shell
✔ What are you trying to accomplish? · Improve typings in my TypeScript files
✔ Where is your tsconfig.json? · tsconfig.json
✔ Which improvements would you like to make? · Add missing property declarations to classes
? Which glob matches files you'd like to convert? … 
  everything in my tsconfig.json
  lib/**/*.{ts,tsx}
  src/**/*.{ts,tsx}
❯ other
```

### Additional Info

Typestat: v0.7.3
node: v20.12.0'

I'm running the tool in this repository

**Comment by JoshuaKGoldberg** [comment]
Yeah that CLI is pretty old. It'd be great to clean it up! I like your suggestions. 🙂

**Comment by JoshuaKGoldberg** [comment]
For parsing libraries: I'd say `node:utils` yeah. No need to dip into userland libraries when the platform does it for us!

+1 to waiting until #1341 is done.
